river/drift/retrain.py

```diff
@@ -39,10 +41,18 @@ class DriftRetrainingClassifier(base.Wrapper, base.Classifier):
 
     """
 
-    def __init__(self, model: base.Classifier, drift_detector: base.DriftAndWarningDetector = None):
+    def __init__(
+        self,
+        model: base.Classifier,
+        drift_detector: base.DriftAndWarningDetector = None,
+        train_in_background: bool = True,
+    ):
         self.model = model
         self.bkg_model = model.clone()
+        self.train_in_background = train_in_background
         self.drift_detector = drift_detector if drift_detector is not None else drift.DDM()
+        if self.train_in_background:
+            self.bkg_model = model.clone()
```

Comment:
No need for this right? You already do this right above (at line 51)

Reply:
Actually a bkg_model must be created only if  self.train_in_background

Reply:
Right!